lms/envs/common.py

```diff
@@ -61,6 +61,7 @@
 }
 
 LMS_ROOT_URL = "http://localhost:8000"
+ENTERPRISE_API_URL = LMS_ROOT_URL + '/enterprise/api/v1/'
```

Comment:
The enrollment API URL sets the default the same way. This value is later overridden for other environments with appropriate values. E.g. in `aws.py`, `devstack_docker.py`, etc. I might be missing the point though, let me know. :)